cocotb/regression.py

```diff
@@ -168,6 +168,11 @@ def _discover_tests() -> Iterable[Test]:
 
         modules = [s.strip() for s in module_str.split(',') if s.strip()]
 
+        tests = []
```

Comment:
```suggestion
        tests = None
```

Reply:
Changed in ccc0ce20dd809c137e93ae20666fd06190fa0f4e.